RawBencher/Program.cs

```diff
@@ -9,6 +8,13 @@
 using System.Reflection;
 using System.Text;
 using System.Threading.Tasks;
+using RawBencher.Benchers;
+using System.Threading;
+using Dapper;
+#if !DNXCORE50

```

Comment:
The project.json-based tooling automatically includes the upper-casified version of the build target as a build symbol. So if your project.json defines a build for dnx451 then DNX451 is defined as a build symbol. This may need revising as the "generations" stuff stabilises: https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md